src/check_python_versions/sources/poetry_pyproject.py

```diff
@@ -83,6 +101,10 @@ def get_supported_python_versions(
         # versions in classifiers.
         return []
 
+    if not isinstance(classifiers, (list, tuple)):
+        warn('The value passed to classifiers is not a list')
```

Comment:
The 'passed to' phrasing makes sense when talking about values passed to Python function calls, which is what happens in a `setup.py`.   For TOML files there are no function calls and no passing, so it should probably say 'The value specified for [tools.poetry] classifiers is not an array'.

Reply:
I haven't used TOML much.  Are there alternative syntaxes?  E.g. could I write something like

```toml
[tools]
poetry = { classifiers: [...] }
```

or 

```toml
[tools.poetry.classifiers]
...
```

instead?  If so maybe the message should say "The value of tools.poetry.classifiers is not an array", or whatever is the common syntax for identifying a piece of a nested data structure in TOML-land.